### PR TITLE
Added YAML configuration file functionality.

### PR DESCRIPTION
**Pull request by bkd705**
## Description

Added YML config functionality to the `src/config.js` file.

## Motivation and Context

This will allow users to keep bundlesize config out of their package.json and put it into a yaml configuration file.

This PR addresses #162.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist:

- My code follows the code style of this project.
- If my change requires a change to the documentation I have updated the documentation accordingly.
- I have read the **CONTRIBUTING** document.
- I created an issue for the Pull Request

### CODE REVIEW

**Comment by siddharthkp** [comment]
Thanks for the PR! Please check the review comments 😄

**Comment by bkd705** [comment]
Hey @siddharthkp I might just be totally blind or missing it, but I don't see any review comments currently.

**Comment by siddharthkp** [comment]
sorry! forgot to hit submit 😅 my bad

File: README.md [hunk 1]
```diff
@@ -111,6 +111,21 @@ example usage:
...
+`bundlesize` also supports configuration using a `.bundlesize.yml` file.
+
+example usage:
+
+```yml
+- path: "./index.js"
+  maxSize: "600B"
+- path: "./src/files.js"
+  maxSize: "600B"
+```
```
[the remark this answers is not in the document]
Comment (bkd705) [reply README.md]: yeah for sure! I'll take a look at this tonight! :P

**Comment by bkd705** [comment]
Should we have just 1 global `gzip` option or a file by file option as well like #172?

**Comment by siddharthkp** [comment]
I think it's more useful to have 1 disable-gzip option. Sadly, there's no way of doing that in the package.json config because it only accepts an array. I plan to change that soon though 😄

**Comment by bkd705** [comment]
Would it be worthwhile to do this and changing the package.json at the same time?

**Comment by palashmon** [comment]
Closing this PR for now because it has not had recent activity. Thank you for your contributions.

**Pull request closed without merging**